pageserver/src/branches.rs

```diff
@@ -17,13 +17,14 @@ use std::{
 use zenith_utils::zid::{ZTenantId, ZTimelineId};
 
 use log::*;
+use zenith_utils::logging;
 use zenith_utils::lsn::Lsn;
 
-use crate::logger;
-use crate::restore_local_repo;
+// use crate::logger;
```

Comment:
Guess this can be removed.
Just to collect all nits in one place: there's also a `pagees` typo a bit below.